packages/terra-form-select/src/single/Frame.jsx

```diff
@@ -530,11 +532,7 @@ class Frame extends React.Component {
           {this.getDisplay(displayId, placeholderId)}
         </div>
         {this.renderToggleButton()}
-        <span
-          aria-live={SharedUtil.isSafari() ? 'polite' : 'off'}
-          className={cx('visually-hidden-component')}
-          ref={this.visuallyHiddenComponent}
-        />
+        {(!this.state.isSelect || this.state.isOpen) && <span aria-live={SharedUtil.isSafari() ? 'polite' : 'off'} className={cx('visually-hidden-component')} ref={this.visuallyHiddenComponent} />}
```

Comment:
I think we need to check only for open state `isOpen` here. since instruction is read twice when drop-down is closed

Reply:
Updated - https://github.com/cerner/terra-core/pull/4002/commits/66ed6d0153f368bddc38757e2c0215d6b5df318b